examples/amm/README.md

```diff
@@ -51,7 +51,7 @@ We use the test-only CLI option `--testing-prng-seed` to make keys deterministic
 explanation.
 
 ```bash
-OWNER_1=e814a7bdae091daf4a110ef5340396998e538c47c6e7d101027a225523985316
+OWNER_1=7136460f0c87ae46f966f898d494c4b40c4ae8c527f4d1c0b1fa0f7cff91d20f
 OWNER_2=453690095cdfe6dbde7fc577e56bb838a7ee7920a72512d4a87748b4e151ed61
```

Comment:
Probably owner 2 has also changed?

Reply:
Ah, right. Good catch. I wonder why the README CI isn't catching it though, since we have graphQL tests 🤔